gremlin-scala/src/test/scala/gremlin/scala/ArrowSyntaxSpec.scala

```diff
@@ -169,14 +170,14 @@ class ArrowSyntaxSpec extends WordSpec with Matchers {
   // }
 
   trait Fixture {
-    implicit val graph = TinkerGraph.open.asScala
-    val paris = graph + "Paris"
+    implicit val graph = TinkerGraph.open.asScala()
+    val paris: Vertex = graph + "Paris"
```

Comment:
Why do you think we should annotate the types here (and in lines 179/180)? 
I'm not completely against it, but IMO it's straightforward to see what they are..

Reply:
Since it's a public member, I think the norm is to add type signature for public members. I think IntelliJ suggests that too. I agree with you that it's a bit redundant and adds a bit of noise. At the same time, I also think having type annotations on public members enforces the contract.
That said, I was going to follow IntelliJ suggestions and add more annotations. But scalac didn't like some of them - it might have to do with either type widening/narrowing or some implicit. I didn't really investigate. I pretty much stopped there :)